code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -168,9 +168,7 @@
 		..()
 
 /mob/living/simple_animal/drone/cogscarab/can_use_guns(obj/item/weapon/gun/G)
-	if(!GLOB.ratvar_awakens)
-		changeNext_move(CLICK_CD_RANGE*4) //about as much delay as an unupgraded kinetic accelerator
-	return TRUE
+	return FALSE
```

Comment:
If ratvar is up a lot of normal rules don't apply because something something UNLIMITED POWER